applications/DEM_application/custom_elements/cluster3D.cpp

```diff
@@ -485,17 +432,14 @@ namespace Kratos {
                     p_continuum_particle_j->mNeighbourElasticContactForces.push_back(zero_vector);
                     p_continuum_particle_j->mNeighbourElasticExtraContactForces.push_back(zero_vector);
                 }
-            }   
-        }               
+            }
+        }
     }
-    
+
     void Cluster3D::Move(const double delta_t, const bool rotation_option, const double force_reduction_factor, const int StepFlag ) {
-        GetTranslationalIntegrationScheme().MoveCluster(this, GetGeometry()[0], delta_t, force_reduction_factor, StepFlag);
+        RigidBodyElement3D::GetTranslationalIntegrationScheme().MoveCluster(this, GetGeometry()[0], delta_t, force_reduction_factor, StepFlag);
         if (rotation_option) {
-            GetRotationalIntegrationScheme().RotateCluster(this, GetGeometry()[0], delta_t, force_reduction_factor, StepFlag);
+            RigidBodyElement3D::GetRotationalIntegrationScheme().RotateCluster(this, GetGeometry()[0], delta_t, force_reduction_factor, StepFlag);
         }
```

Comment:
I see that we are using methods from RigidBodyElement3D, can't we use the full function Move here? open to discussion.

Reply:
Changed! More functions have been removed from Cluster3D element.